po/pt_BR.po

```diff
@@ -173,72 +187,79 @@ msgid ""
 "ability to manually adjust line matching and directly edit files. Diffuse can\n"
 "also retrieve revisions of files from several VCSs for comparison and merging."
 msgstr ""
+"O ‘Diffuse’ é uma ferramenta por interface gráfica que permite comparar e\n"
+"mesclar os arquivos de texto, comparar uma quantidade arbitrária de arquivos\n"
+"lado a lado e oferece aos usuários a capacidade de ajustar manualmente a\n"
+"correspondência das linha e editar os arquivos diretamente, possibilita\n"
+"recuperar as revisões feitas com as ferramentas ‘Bazaar’, ‘CVS’, ‘Darcs’,\n"
+"‘Git’, ‘Mercurial’, ‘Monotone’, ‘RCS’, ‘Subversion’ e ‘SVK’."
```

Comment:
change: you seem to have copy/pasted the same paragraph here too :face_with_diagonal_mouth: